app/views/artists/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class='tiles-container'>
+ <%= render partial: 'tile',
+   locals: { artist_public_data: @artist_public_data, context: :show } %>
+
+  <div class='tile tile-no-background'>
+    <dl>
+      <dt>Born</dt>
```

Comment:
Since we're pulling the first paragraph of data for artists from Wikipedia, the birth and deceased dates will look redundant. I think we can remove Born/Died from the page.

You can run `rake bpac:populate_artists_from_wikipedia` to load Wikipedia data locally.

![screen shot 2015-08-01 at 5 36 13 pm](https://cloud.githubusercontent.com/assets/772925/9023701/921d98ba-3874-11e5-8bc1-080ea463ea44.png)